proto/rill/admin/v1/api.proto

```diff
@@ -2640,6 +2640,7 @@ message Bookmark {
 
 message ServiceToken {
   string id = 1;
+  string token_prefix = 4;
```

Comment:
I suggest just naming it `prefix`, not `token_prefix`.

The type serves as scope to provide context. Like `ServiceToken.prefix` is probably clear enough. Similar to how the other fields are not `ServiceToken.token_expires_on`, but just `ServiceToken.expires_on`.